python-package/brainflow/board_shim.py

```diff
@@ -46,6 +47,7 @@ class BoardIds(enum.IntEnum):
     MUSE_S_BLED_BOARD = 21  #:
     MUSE_2_BLED_BOARD = 22  #:
     CROWN_BOARD = 23  #:
+
```

Comment:
new line here is not needed, it seems like in python you have some autoformatter, I am fine with it but here new line looks bizarre